src/containers/discovery-node-selection-preview/DiscoveryNodeSelectionPreview.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+
+import { Button, ButtonType } from '@audius/stems'
+import { ChangeEvent } from 'react-slick/node_modules/@types/react'
+
+import { usePreviewHotkey } from 'hooks/useHotkey'
+
+import styles from './DiscoveryNodeSelectionPreview.module.css'
+
+const DISCOVERY_NODE_SELECTION_ENABLE_KEY =
+  'enable-discovery-node-selection-preview'
+
+const DiscoveryNodeSelectionPreview = () => {
+  const isEnabled = usePreviewHotkey(
+    68 /* d */,
+    DISCOVERY_NODE_SELECTION_ENABLE_KEY
+  )
+  const [endpoint, setEndpoint] = useState('')
+  const [success, setSuccess] = useState(false)
+  const [error, setError] = useState(false)
+
+  const handleDiscoveryNodeSelection = async () => {
+    const healthyEndpoints = (
+      await (await fetch('https://api.audius.co')).json()
```

Comment:
i dont want to vendor api.audius.co like this into the client. libs has the capacity to go off of the chain to determine this -- but i actually think this tool would be most useful if we *dont* cross reference it for health. it would allow you to specific any node (even during dev)